src/main/kotlin/no/nav/familie/oppdrag/simulering/SimulerBeregningResponseMapper.kt

```diff
@@ -0,0 +1,46 @@
+package no.nav.familie.oppdrag.simulering
+
+import no.nav.familie.kontrakter.felles.oppdrag.RestSimulerResultat
+import no.nav.system.os.entiteter.beregningskjema.BeregningsPeriode
+import no.nav.system.os.tjenester.simulerfpservice.simulerfpservicegrensesnitt.SimulerBeregningResponse
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+fun SimulerBeregningResponse.toRestSimulerResult(
+        dato: LocalDate = LocalDate.now()): RestSimulerResultat {
+
+    val totalEtterbetalingsBeløp =
+            this.response?.simulering?.beregningsPeriode?.sumBy { finnEtterbetalingPerPeriode(it, dato) }
+            ?: 0
+
+    return RestSimulerResultat(etterbetaling = totalEtterbetalingsBeløp)
+}
+
+private fun finnEtterbetalingPerPeriode(beregningsPeriode: BeregningsPeriode, dato: LocalDate): Int {
+    // Fremtidige perioder gir ingen etterbetaling.
+    val datoFraPeriode = LocalDate.parse(beregningsPeriode.periodeFom, DateTimeFormatter.ISO_DATE)
+    if (datoFraPeriode.month > dato.month) return 0
+
+    val stoppNivaBA =
+            beregningsPeriode.beregningStoppnivaa.filter { it.kodeFagomraade?.trim() == "BA" }
```

Comment:
Vi vil helst ikke ha kode som er spesifikk for én ytelse i familie-oppdrag. Kan du ta inn fagområde som en parameter her i stedet for å hardkode det til BA?